externs/olx.js

```diff
@@ -1110,6 +1111,12 @@ olx.control.FullScreenOptions.prototype.keys;
  */
 olx.control.FullScreenOptions.prototype.target;
 
+/**
+ * Source element for FullScreen. When not provided the `map.target` DOM element will be used as FullScreen element.
```

Comment:
The spec spells this `Fullscreen`.  The document property is spelled `fullscreenElement`.  In any case, I don't think this should be "element for FullScreen."  How about something like this:

> The element to be displayed fullscreen.  When not provided, the element containing the map viewport will be displayed fullscreen.